test/lensZip.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * test/lensZip.js
+ */
+
+const chai = require('chai');
+const path = require('path');
+const expect = chai.expect;
+const fs = require('fs-extra');
+const childProcess = require('child_process');
+const unzip = require('unzipper');
+
+describe('lens zip test >', function () {
+  this.timeout(5000);
+
+  it('lens zip >', (done) => {
```

Comment:
our convention is for a "describe" subject line to end with ">" but not the "it" subject line